installer/Octopus.Tentacle.Installer/Product.wxs

```diff
@@ -140,16 +140,11 @@
     <!--
     Harden installation directory
     -->
-    <CustomAction Id="HardenEXE"
-                  Directory="INSTALLLOCATION"
-                  ExeCommand="&quot;[INSTALLLOCATION]Octopus.Tentacle.Hardener.exe&quot;"
-                  Execute="deferred"
-                  Return="ignore"
-                  Impersonate="no"
-                 />
-
+    <CustomAction Id="SetExecProp" Property="WixQuietExecCmdLine" Value='"[POWERSHELLEXE]" -NoProfile -NonInteractive -InputFormat None -File "[INSTALLLOCATION]Harden-InstallationDirectory.ps1"'></CustomAction>
```

Comment:
The original command was:
```
[POWERSHELLEXE]" -NoProfile -NonInteractive -InputFormat None -ExecutionPolicy Bypass -File "[INSTALLLOCATION]Harden-InstallationDirectory.ps1"
```
But I don't think bypassing the execution policy is what we want to do so I've removed it.

Reply:
I did some more testing and it looks like `-ExecutionPolicy Bypass` is needed. Maybe it's something to do with Wix - even though the policy is set to RemoteSigned on my laptop, when I write the output to a log it says running scripts is disabled.